xc/device.ml

```diff
@@ -2473,8 +2506,20 @@ module Dm = struct
         )
     | _ -> failwith "Unsupported vGPU configuration"
 
-  let __start (task: Xenops_task.task_handle) ~xs ~dm ?(timeout = !Xenopsd.qemu_dm_ready_timeout) l info domid =
-    debug "Device.Dm.start domid=%d args: [%s]" domid (String.concat " " l);
+
+  type action = Start | Restore | StartVNC
+
+  let __start (task: Xenops_task.task_handle)
+      ~xs ~dm ?(timeout = !Xenopsd.qemu_dm_ready_timeout) action info domid =
+
+    let args = match action with
+      | Start    -> qemu_args ~xs ~dm info false domid
```

Comment:
Please use the boolean as named argument `domid_for_vnc`

Reply:
This is not `domid_for_vnc` but the `restore` parameter which has no label.

Reply:
:P Oh I see, sorry I had misread the function signature